src/components/common/NavBar.js

```diff
@@ -28,18 +29,18 @@ function NavBar(props) {
   }
 
   const backBtn = navBarType === "module" ? (
-    <div className="nav-back-btn">
+    <div className="nav-back-btn" onClick={toggleSideBar}>
       <svg className="hamburger-icon">
         <use xlinkHref="/website-icons.svg#hamburger-icon"></use>
       </svg>
-      <a href="/">3. Linked List Insertion</a>
+      <p>{SubModuleTitle}</p>
     </div>
   ) : null;
 
   return (
     <div className={containerClass}>
       {backBtn}
-      <h1><Link to="/">Node Warrior</Link></h1>
+      <h1><Link to="/">{brandTitle}</Link></h1>
```

Comment:
Same note from above, I think it would be better just to state the name of the website rather than make it a prop.